### DISCUSSION TITLE
Dynamic breadcrumbs

### DISCUSSION THREAD

**Discussion by sayyad84** [OrchardCMS/OrchardCore#12652] [Q&A]
Is it possible to generate a multi-level breadcrumbs on each page automatically and dynamically?

**Comment by hishamco** [comment]
This https://orcharddojo.net/blog/how-to-add-a-breadcrumb-menu-in-orchard-core-orchard-nuggets could be a good start

**Reply by sayyad84** [reply]
Thanks for your reply
But my menu items are ContentMenuItemPart instead of LinkMenuItemPart and the solution you mentioned does not work for them. How can I access the LinkMenuItemPart -> AutoRoutePart-> Url so that they can be compared with currentRelativeUrl?

**Comment by sayyad84** [comment]
Thanks for your reply
I solved the problem myself

**Reply by hishamco** [reply]
Hope the above sample was helpful
